modules/ctl/src/main/scala/low/git.scala

```diff
@@ -0,0 +1,58 @@
+package gem.ctl
+package low
+
+import scalaz._, Scalaz._
+import scala.util.matching.Regex
+
+import gem.ctl.free.ctl._
+import io._
+
+/** Low-level constructors for `CtlIO` operations related to git. */
+object git {
+
+  case class Commit(hash: String)
+  object Commit {
+    implicit val OrderCommit: Order[Commit] = Order.orderBy(_.hash)
+  }
+
+  case class Tag(tag: String)
+
+  // Yields most recent tag matching the given pattern, if any
+  def mostRecentTag(pattern: Regex): CtlIO[Option[Tag]] =
+    shell("git", "tag").require {
```

Comment:
I see your point. This is a problem also when the shell tools evolve and their syntax change. It would probably be good to say, this works with this version at least.